syntax = "proto2";
package DBProxyServer;

option cc_generic_services = true;

message MySqlRequest {
    required string dbname = 1;
    required string option = 2;
    optional string message = 3;
};

message MySqlResponse {
    required int32 status = 1;
    optional string message = 2;
};

service MySqlService {
    rpc LoginOrRegister(MySqlRequest) returns (MySqlResponse);
};

